scripts/compare_noupdate_xml_records.py

```diff
@@ -185,12 +200,13 @@ def main(argv=None):
         if len(element):
             data.append(element)
 
-    openerp = etree.Element("openerp")
-    openerp.append(data)
-    document = etree.ElementTree(openerp)
+    odoo = etree.Element("odoo")
+    odoo.append(data)
```

Comment:
Oh, but I was talking about the output. You are still producing files with a data node, aren't you?